app/src/test/java/com/example/android/architecture/blueprints/todoapp/data/source/DefaultTasksRepositoryTest.kt

```diff
@@ -60,7 +67,7 @@ class DefaultTasksRepositoryTest {
     fun getTasks_emptyRepositoryAndUninitializedCache() = runBlockingTest {
```

Comment:
Since you've added MainCoroutineRule, all of these runBlockingTest should be mainCoroutineRule.runBlockingTest